<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>siginup</class>
 <widget class="QWidget" name="siginup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191, 215, 234);
</string>
  </property>
  <widget class="QPushButton" name="pbn_cancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 20%;
color:rgb(245, 223, 187);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_submit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 20%;
color:rgb(245, 223, 187);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15px;
border-color : rgb(98, 98, 98);</string>
   </property>
   <property name="title">
    <string>Form</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>300</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>confirm password :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="conf_pass">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>Password :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>BirthDate :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="led_email">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="led_name">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>PhoneNumber :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="led_phonenumb">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="led_pass">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>UserName :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 48, 73);</string>
       </property>
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
